src/routes/__root.tsx

```
@@ -50,18 +51,20 @@ function RootComponent() {
 
-function RootDocument({ 
-    children 
-}: { 
-    children: ReactNode 
+function RootDocument({
+  children
+}: {
+  children: ReactNode
 }) {
   return (
     <html>
```

Biome (2.1.2)

[error] Provide a lang attribute when using the html element.

Setting a lang attribute on HTML document elements configures the languageused by screen readers when no user default is specified.

(lint/a11y/useHtmlLang)